lib/checks/aria/no-implicit-explicit-label.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "no-implicit-explicit-label",
+	"evaluate": "no-implicit-explicit-label.js",
+	"metadata": {
+		"impact": "moderate",
+		"messages": {
+			"fail": "Element with ARIA role {{=it.data}} must have label",
```

Comment:
This check should never call out a failure. This rules come back either passed, or with one of the following two messages:

Failed:
```md
Fix at least one of the following:
- ... aria-labelledby ...
- ... aria-label ...
- ... non-empty title ...
```

Incomplete:
```md
Fix the following:
- Check that the <label> does not need be part of the ARIA {=it.data=} field's name 
```